autoload/ale/completion.vim

```diff
@@ -509,17 +509,18 @@ function! ale#completion#GetCompletions() abort
         return
     endif
 
-    call ale#completion#AlwaysGetCompletions()
+    call ale#completion#AlwaysGetCompletions(1)
 endfunction
 
 " This function can be used to manually trigger autocomplete, even when
 " g:ale_completion_enabled is set to false
-function! ale#completion#AlwaysGetCompletions() abort
+function! ale#completion#AlwaysGetCompletions(...) abort
```

Comment:
This function isn't part of ALE's public API, so just add a new argument here and update the command. This is why I asked for the command to be part of the public API, but not the function. :wink: